README.md

```diff
@@ -27,70 +27,65 @@ In the [`bkc`](https://github.com/intel/ccc-linux-guest-hardening/tree/master/bk
 
-- `python3`
-- `python3-venv`
-
-~~~
-sudo apt-get install python3 python3-venv
-~~~
+- **Know your Kernel:** Working knowledge of Linux console, kernel build and boot,
+  and an idea of the kernel version and feature you want to test.
 
 ## Installation
 
-Clone this repo to a new directory, e.g. `~/tdx`:
-
-```shell
-git clone https://github.com/intel/ccc-linux-guest-hardening ~/tdx
-cd ~/tdx
-```
-
-We use Ansible playbooks to support local or remote installation.
-
-### Local
-
-- installation directory: `<repo_root>/`
+### The installation and the fuzzing runtime requires Python3 and the virtual environment package:
 
-Run the deployment with:
 ~~~
-make deploy
+sudo apt-get install python3 python3-venv
 ~~~
 
-You will be prompted for your root password.
-If you are using a _passwordless sudo_ setup, just skip this by pressing enter.
+### Clone this repo to a new top-level workspace and install using `make deploy`:
 
-### Remote
+  ```bash
+  git clone https://github.com/intel/ccc-linux-guest-hardening ~/cocofuzz
+  cd ~/cocofuzz
+  make deploy
+  ```
 
-- installation directory: `$HOME/ccc`
+**Note:** The installation uses [Ansible](https://docs.ansible.com/ansible/latest/).
+The main system modification is to install a patched host kernel (`.deb` package)
+and fixing the `grub` config to make it boot. Ansible will also add the current
+user to group `kvm` and pull in a few build dependencies and tools via `apt`. 
+The rest of the stack consists of userspace tools and scripts which are only 
+available in a local Python virtual environment.
 
-You will have to update the `deploy/inventory` file to describe your nodes, according to [Ansible's inventory guide](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html).
-Make sure to **remove** the first line:
+### If not yet done, reboot to launch the kAFL/SDV emulation kernel:
 
-~~~
-localhost ansible_connection=local
-~~~
+```bash
+uname -a
+# Linux tdx-fuzz0 5.6.0-rc1-tdfl+ #15 SMP Wed May 25 02:23:44 CEST 2022 x86_64 x86_64 x86_64 GNU/Linux
+```
 
-And run the deployment:
+```bash
+dmesg|grep KVM-PT
+# [KVM-PT] Info:  CPU is supported!
+# [KVM-PT] Info:  LVT PMI handler registrated!
+```
 
-~~~
-make deploy
-~~~
 
-Note: if your nodes require a proxy setup, update the `group_vars/all.yml`.
 
 ## Activate the environment
 
 When the installation is complete, you will find several tools and scripts
 inside the installation directory of the target system.
 
+The environment defines various default paths used by multiple layers of
+scripts. Go take a look. The script also sets `MAKEFLAGS="-j$(nproc)"` as a global
+default for parallel builds. Watch out for effects of this.
+
 All subsequent steps assume that you have activated the installation environment.
 This is done either by sourcing the `env.sh` script, or by typing `make env`,
 which launches a sub-shell that makes it easier to exit and switch environments:
```

Comment:
The second sentence here can be removed, we can just mention `make env` as code example. But it should be before the previous paragraph about MAKEFLAGS etc.

Reply:
Removed the second sentence and reordered the two related paragraphs.